### PR TITLE
Feature/add multiple config

### PR DESCRIPTION
**Pull request by TheNeikos**
Solves #223

### CODE REVIEW

**Comment by TheNeikos** [comment]
@matthiasbeyer should this repo continue to use the dot version?

File: src/config.rs [hunk 1]
```diff
@@ -108,16 +108,24 @@ impl std::str::FromStr for EditFormat {
...
+        };
+
+        if check_path.exists() {
+            changelog_config_path = Some(check_path);
+            break;
+        }
     }
 
+    let Some(changelog_config_path) = changelog_config_path else {
+            miette::bail!(Error::ConfigDoesNotExist)
```
Comment (matthiasbeyer) [inline src/config.rs:126]: Indention seems wrong here.

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]

**Comment by matthiasbeyer** [comment]
> @matthiasbeyer should this repo continue to use the dot version?

No, it should use the defaults.

File: src/main.rs [hunk 2]
```diff
@@ -118,7 +118,7 @@ fn init(repo_workdir_path: PathBuf) -> miette::Result<()> {
         .create(true)
         .append(false)
         .write(true)
-        .open(repo_workdir_path.join(crate::config::CONFIG_FILE_NAME))
+        .open(repo_workdir_path.join(crate::config::CONFIG_FILE_NAMES[1]))
```
Comment (matthiasbeyer) [inline src/main.rs:121]: IMO we should have a global `CONFIG_FILE_DEFAULT_NAME` which does that indexing into the `CONFIG_FILE_NAMES` array...

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]
Just one small change please.

**Review by matthiasbeyer — APPROVED** [review]

**Comment by matthiasbeyer** [comment]
bors merge

**Pull request merged**